woocommerce/woocommerce-android
# [Themes] - Update the error screen UI

Resolves #10443. This PR updates the theme picker error screen to align with iOS. There's also a new Retry button.

[Screen_recording_20231220_002925.webm](https://github.com/woocommerce/woocommerce-android/assets/1522856/97d09772-5032-4b12-acbc-742808184768)

**To test:**
Try the following both with an existing site and in the creation flow:

1. Turn on Airplane mode
2. Go to Themes
3. Notice there's a new error message with a Retry button
4. Disable the Airplane mode
5. Tap on the button
6. Notice the themes are loaded
